.evergreen/config.in.yml

```diff
@@ -200,6 +200,7 @@ functions:
       type: test
       params:
         working_dir: "src"
+        add_expansions_to_env: true
```

Comment:
just curious - how come this addition is necessary?

Reply:
I have undone this, it makes sense to only change what we need.